unzip_content.py

```diff
@@ -101,14 +101,13 @@ def get_icon_path(linkJSON):
     # For now, we need to convert specific locales to personalities,
     # including duplication of en-us as both default and Global,
     # until the CMS is reworked
-    locales = ['en-us', 'en-us', 'es-gt', 'pt-br', 'zh-hans']
+    locales = ['en-us', 'en-us', 'es-gt', 'pt-br', 'zh_CN']
```

Comment:
Ah, never mind.  I see now that link_locales is a separate variable with 'zh-hans'.